hermes-client/src/main/java/pl/allegro/tech/hermes/client/HermesClient.java

```diff
@@ -128,4 +142,30 @@ public void close(long pollInterval, long timeout) throws InterruptedException,
         }
     }
 
+    private void handleMaxRetriesExceeded(ExecutionCompletedEvent<HermesResponse> event) {
+        if (event.getResult().isSuccess()) {
```

Comment:
Can getResult() be null?

Reply:
it can't be, here we always have the result, each sending operation ends with an appropriate result with an error or success